### ISSUE TITLE
Carthage error 65

### ISSUE THREAD

**Issue by alekzernov** [appsquickly/Typhoon#587]
When installed through Carthage
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/alekzernov/Downloads/enegyhouserev2.0/Carthage/Checkouts/Typhoon/Typhoon.xcodeproj -scheme Typhoon -configuration Release -derivedDataPath /Users/alekzernov/Library/Caches/org.carthage.CarthageKit/DerivedData/9.4_9F1027a/Typhoon/4.0.5 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=1D1D025C-9F81-4183-B247-1F69285384B6 -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/alekzernov/Downloads/enegyhouserev2.0/Carthage/Checkouts/Typhoon)
help me please

**Comment by alekzernov** [comment]
fatal error: 'OCMockitoIOS/OCMockitoIOS.h' file not found
        #import <OCMockitoIOS/OCMockitoIOS.h>
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~

** BUILD FAILED **

**Comment by thomaskolmans** [comment]
Yes, I had the exact same issue. I solved it for now by specifying the version as this seems to be a 4.0.5 issue.

`github "appsquickly/Typhoon" "4.0.4"` solved it for me.

**Comment by vitorhugomagalhaes** [comment]
Any chance to get this fixed. It's always good to use the latest release. Thanks

**Comment by hydra1983** [comment]
```bash
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/edison/workspace/weiguan/project/gitlab/wg-dev/wg-app-ios/Carthage/Checkouts/Typhoon/Typhoon.xcodeproj -scheme Typhoon -configuration Release -derivedDataPath /Users/edison/Library/Caches/org.carthage.CarthageKit/DerivedData/9.4.1_9F2000/Typhoon/4.0.6 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=5FE46D7A-0E86-44A9-A986-16425353E16F -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/edison/workspace/weiguan/project/gitlab/wg-dev/wg-app-ios/Carthage/Checkouts/Typhoon)
```

Same as 4.0.6 :(

**Comment by alexgarbarev** [comment]
Can anyone help me with this issue? I'm not very familiar with Carthage.
